middleware/file/README.md

```diff
@@ -48,3 +48,8 @@ file example.org.signed example.org {
     transfer to 10.240.1.1
 }
 ~~~
+
+## Interaction with other middlewares
```

Comment:
Please drop this from this README. We should make that clear in the README of *cache*